### PR TITLE
formula: add more API docs for directories.

### PR DESCRIPTION
**Pull request by mikemcquaid**
CC @Homebrew/owners for thoughts/review.

### CODE REVIEW

File: Library/Homebrew/formula.rb [hunk 1]
```diff
@@ -272,41 +274,140 @@ def installed_version
...
   def info;    share+'info'     end
+
+  # The directory where the formula's libraries should be installed.
+  # This is symlinked into `HOMEBREW_PREFIX` after installation or with
+  # `brew link` for formulae that are not keg-only.
   def lib;     prefix+'lib'     end
+
+  # The directory where the formula's binaries should be installed.
+  # This is not symlinked into `HOMEBREW_PREFIX`.
+  # It is also commonly used to install files that we do not which to be
```
Comment (bfontaine) [inline Library/Homebrew/formula.rb]: > that we do not *which* to be

Is it a typo for “wish”?

File: Library/Homebrew/formula.rb [hunk 2]
```diff
@@ -272,41 +274,140 @@ def installed_version
...
+
+  # The directory where the formula's man8 pages should be installed.
+  # This is symlinked into `HOMEBREW_PREFIX` after installation or with
+  # `brew link` for formulae that are not keg-only.
   def man8;    man+'man8'       end
+
+  # The directory where the formula's `sbin` binaries should be installed.
+  # This is symlinked into `HOMEBREW_PREFIX` after installation or with
+  # `brew link` for formulae that are not keg-only.
+  # Generally we trying to migrate these to {#bin} instead.
```
Comment (bfontaine) [inline Library/Homebrew/formula.rb]: Shouldn’t it be “Generally we try to”?

File: Library/Homebrew/formula.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Homebrew/formula.rb]: :+1:

File: Library/Homebrew/formula.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Homebrew/formula.rb]: :+1:

**Pull request closed without merging**